src/pages/benefits/benefitFormUI/BenefitApplicationForm.tsx

```
@@ -48,6 +76,8 @@ const BenefitApplicationForm: React.FC = () => {
   const [disableSubmit, setDisableSubmit] = useState(false);
   const [uiSchema, setUiSchema] = useState({});
   const [reviewerComment, setReviewerComment] = useState<string | null>(null);
+  const [documentFieldNames, setDocumentFieldNames] = useState<string[]>([]);
+  const [docsArray, setDocsArray] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)